src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -1676,6 +1708,24 @@ public FrameType getType() {
             return FrameType.CATCH_FRAME;
         }
 
+        @Override
+        protected AbstractFrame getIfContains(DetailAST nameToFind, boolean lookForMethod) {
```

Comment:
>DetailAST nameToFind

"Name" vs ast confuse my brain. Can we have different name?

Reply:
`identToFind`? In the end, `IDENT`s are being compared.